includes/theme-compat/header.php

```diff
@@ -14,43 +14,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title><?php echo $meta_title; ?></title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <?php echo '<link rel="stylesheet" href="' . BASE_URL . '/assets/css/master.css">'; ?>
+	<?php echo '<link rel="stylesheet" href="' . BASE_URL . '/assets/css/master.css">'; ?>
     <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
 <div class="site-wrapper">
     <header class="site-header">
         <div class="row site-header-inner">
             <div id="header_logo">
-                <a href="/" title="logo"> <img class="site-header-branding" src="/img/logo.png" alt="logo"
-                                               width="300" height="80"> </a>
+                <a href="/" title="logo"> <img class="site-header-branding" src="/img/logo.png" alt="logo" width="300"
+                                               height="80"></a>
             </div>
             <nav class="site-header-navigation">
                 <ul class="menu">
-                    <li class="menu-item <?php if (isset($page_name) && $page_name == 'home') { ?> current-menu-item <?php } ?>">
+                    <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'home' ) { ?> current-menu-item <?php } ?>">
                         <a href="/index.php">Home</a>
                     </li>
-                    <?php if (isset($_SESSION['id_user'])) { ?>
-                        <?php if (isset($_SESSION['is_admin']) && $_SESSION['is_admin']) { ?>
-                            <li class="menu-item <?php if (isset($page_name) && $page_name == 'dashboard') { ?> current-menu-item <?php } ?> ">
+					<?php if ( isset( $_SESSION['id_user'] ) ) { ?>
+						<?php if ( isset( $_SESSION['is_admin'] ) && $_SESSION['is_admin'] ) { ?>
+                            <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'dashboard' ) { ?> current-menu-item <?php } ?> ">
                                 <a href="/dashboard.php">Dashboard</a>
                             </li>
-                        <?php } ?>
-                        <li class="menu-item <?php if (isset($page_name) && $page_name == 'profile') { ?> current-menu-item <?php } ?>">
+						<?php } ?>
+                        <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'profile' ) { ?> current-menu-item <?php } ?>">
```

Comment:
You shouldn't be using inline IF structures because this is super confusing and hard to read. Instead you should use a ternary operator.

```suggestion
                        <li class="menu-item <?php echo isset( $page_name ) && 'profile === '$page_name ? 'current-menu-item' : ''; ?>">
```

Reply:
This is valid for all other inline IFs